examples/notebooks/getting-started/04 - Querying.ipynb

```diff
@@ -435,11 +449,27 @@
    "source": [
     "## Graph traversing\n",
     "\n",
-    "SPARQL is used to traverse the graph. The user can provide simplified queries: no prefixes, no compacted URIs, nor full URIs is required, only the term (property) available in the Model to the user.  Refer to the `11 - Modeling.ipynb` notebook to learn about Templates.\n",
+    "SPARQL is used as a query language.\n",
     "\n",
+    "A smart rewriting of the query lets users write simplified queries using only the property and type names part of the templates. No need to provide prefixes declaration, to prefix property and type names, nor to use full IRIs.\n",
```

Comment:
To me, except for the 'smart' aspect, this says the same thing but using a negative formulation rather than a positive ('only' vs 'without'). I have switched to the negative formulation.

Why using 'URI' instead of 'IRI' which what is used in the SPARQL vocabulary though?

Reply:
Good.

>Why using 'URI' instead of 'IRI' which what is used in the SPARQL vocabulary though?
IRI is perfect.

About 'smart' now:

While the SPARQL rewriting certainly simplifies the search part, I personally found difficult to justify or prove what makes something 'smart'. But I can live with it though.